hooks/useLinkPreview.ts

```
@@ -1,4 +1,7 @@
+import clientUploadToArweave from "@/lib/arweave/clientUploadToArweave";
+import getBlob from "@/lib/getBlob";
 import { useQuery } from "@tanstack/react-query";
+import { url } from "inspector";
```

ESLint

[error] 'url' is defined but never used.

(@typescript-eslint/no-unused-vars)